src/site/content/en/blog/when-to-use-local-https/index.md

```diff
@@ -76,7 +78,7 @@ You need to use HTTPS for local development in the following cases:
 
 ## How to use HTTPS for local development
 
-If you need to use HTTPS for local development, head over to [How to use HTTPS for local development](/how-to-use-local-https).
+If you need to use HTTPS for local development, head over to **[How to use HTTPS for local development](/how-to-use-local-https)**.
```

Comment:
```suggestion
If you need to use HTTPS for local development, head over to [How to use HTTPS for local development](/how-to-use-local-https).
```

Reply:
Same as above, I don't think this does much except making our link styling inconsistent.